zebra-chain/src/block/tests.rs

```diff
@@ -210,6 +211,11 @@ proptest! {
 
         prop_assert_eq![header, other_header];
     }
+}
+
+proptest! {
+    // The block roundtrip test can be really slow
+    #![proptest_config(Config::with_cases(16))]
```

Comment:
Where does the value '16' come from?

Reply:
The largest power of two which executes in under 10 seconds on my machine.